sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -70,6 +70,7 @@ export default defineComponent({
     const LOCATIONS_DROPDOWN_KEY = "selectedLocation";
     const PROGRAMS_DROPDOWN_KEY = "selectedProgram";
     const OFFERINGS_DROPDOWN_KEY = "selectedOffering";
+    const PROGRAM_INTENSITY_DROPDOWN_KEY = "howWillYouBeAttendingTheProgram";
```

Comment:
IMO the variable must be named as `OFFERING_INTENSITY_DROPDOWN_KEY`

Reply:
RHS says `howWillYouBeAttendingTheProgram` and that is the reason I named it as `PROGRAM_INTENSITY_DROPDOWN_KEY`. It is the same variable impacting both program and offering intensities. Let me know if you still want me to change it.

Reply:
Changing it as per the discussion today.